amethyst_renderer/examples/material.rs

```diff
@@ -103,7 +107,10 @@ fn main() {
             _ => (),
         });
 
-        renderer.draw(&scene, &pipe, delta);
+        let data = List::new().push(List::new().push(
```

Comment:
It is `List` of `List` because `Pass`es are in `List` in `Stage`s which are in `List` in `Pipeline`. The data for `Pass`es is arranged the same way.
Although examples from amethyst_renderer crate aren't refactored for latest changes in `Pipeline`.

Reply:
It's a heterogenous list.

Reply:
@Rhuagh yeah. Thank you. I forgot to mention that.